qiskit_nature/circuit/library/ansatzes/utils/fermionic_excitation_generator.py

```diff
@@ -106,6 +96,12 @@ def generate_fermionic_excitations(
     """Generates all possible excitations with the given number of excitations for the specified
     number of particles distributed among the given number of spin orbitals.
 
+    The method must be called for each type of excitation (singles, doubles, etc.) that is to be
+    considered in the Ansatz. Excitations will be produced based on an initial `Hartree-Fock`
+    occupation by default unless `generalised` is set to `True`. Therefore, the excitations are
```

Comment:
```suggestion
    occupation by default unless `generalized` is set to `True`. Therefore, the excitations are
```
I know.. British English :smile: But in this case it's important because here we're actually referring to the argument name.

Reply:
I'm also not sure about `Therefore` being used here. It should be stressed that the excitations are only **in**dependent from the number of particles if and only if `generalized=True`. The `Therefore` reads a bit differently imo